lib/m2ts/probe.js

```diff
@@ -121,6 +121,11 @@ var parsePesTime = function(packet) {
 
   var offset = 4 + parseAdaptionField(packet);
 
+  if (offset >= packet.byteLength) {
```

Comment:
Although the note below is a succinct description, it might be useful to expand a bit and say that if a packet contains data, it will have pts/dts values. If a packet doesn't have data, and is just stuffing bytes (0xFF), then the adaptation field length will expand to fill the rest of the packet, therefore the offset will reach the end of the packet.